app/controller/admin.php

```diff
@@ -518,7 +518,14 @@ public function group_edit(\Base $f3, array $params)
 
         $members = new \Model\Custom("user_group_user");
         $f3->set("members", $members->find(array("group_id = ? AND deleted_date IS NULL", $group->id)));
-
+        
+        $to_user = new \Model\User;
+        $members2 = new \Model\Custom("user_group_user");
+        $to_user->load(array('email = ? AND deleted_date IS NULL', 'contact@interfast-intl.be'));
```

Comment:
You probably don't want the email address hard-coded here.

Reply:
Indeed, I have to review this part, I will come soon with the correct code. I think there is something to change with the $members2 variable.

Reply:
Those line should be removed, they were part of a test. I will doublecheck this week and update my code.